testsuite/run_sets/build_validation_prepare_clm_filters.yml

```diff
@@ -0,0 +1,7 @@
+# This file describes the order of features in a Build Validation test suite run.
+
```

Comment:
Are you sure this is only for RHEL?
I would say this is for any RH-like system.

Reply:
it's for all RH-like, yes